code/game/machinery/camera/camera.dm

```diff
@@ -80,14 +80,13 @@
 	. = ..()
 	update_icon()
 	if(!c_tag)
-		if(isturf(loc))
-			var/area/A = get_area(src)
-			if(A)
-				for(var/obj/machinery/camera/C in A)
-					if(C == src) continue
-					if(C.number)
-						number = max(number, C.number+1)
-				c_tag = "[A.name][number == 1 ? "" : " #[number]"]"
+		var/area/A = get_area(src)
+		if(isturf(loc) && A)
```

Comment:
Doesn't `A` being truthy imply loc is a turf?

Reply:
Uh, no. The loc being a turf *should* imply that A is truthy, though, but that's the other way around.
get_area works even if the direct loc of the object isn't a turf, which is what isturf(loc) checks.

Reply:
Oh, yep, I follow.